### PR TITLE
Update angular 9.1

### PR DESCRIPTION
**Pull request by jdroenner**
update to angular 9.1
update guide: https://blog.angular.io/version-9-1-of-angular-now-available-typescript-3-8-faster-builds-and-more-eb292f989428

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -98,7 +105,6 @@
         "no-switch-case-fall-through": true,
         "no-trailing-whitespace": true,
         "no-unused-expression": true,
-        "no-use-before-declare": true,
```
Comment (ChristianBeilschmidt) [inline tslint.json:101]: why is this deleted?

**Comment by jdroenner** [comment]
migration did it. i guess its default?
